lib/cloud/aws/aws.go

```diff
@@ -145,6 +145,21 @@ func AuroraMySQLVersion(cluster *rds.DBCluster) string {
 	return version
 }
 
+// IsDocumentDBClusterSupported checks whether IAM authentication is supported
+// for this DocumentDB cluster.
+//
+// https://docs.aws.amazon.com/documentdb/latest/developerguide/iam-identity-auth.html
+func IsDocumentDBClusterSupported(cluster *rds.DBCluster) bool {
+	ver, err := semver.NewVersion(aws.StringValue(cluster.EngineVersion))
+	if err != nil {
+		log.Errorf("Failed to parse DocumentDB engine version: %s", aws.StringValue(cluster.EngineVersion))
+		return false
+	}
+
+	minIAMSupportedVer := semver.New("5.0.0")
+	return !ver.LessThan(*minIAMSupportedVer)
```

Comment:
no need to parse a string
```suggestion
	return !ver.LessThan(semver.Version{Major: 5})
```